server/pxf-service/src/scripts/pxf-env-default.sh

```diff
@@ -73,3 +73,6 @@ export PXF_OOM_KILL=${PXF_OOM_KILL:-true}
 
 # Dump heap on OutOfMemoryError, set to dump path to enable
 # export PXF_OOM_DUMP_PATH=
+
+# Login user for hadoop, defaults to the logged in user in the OS
+# export HADOOP_USER_NAME=
```

Comment:
we advise people not to touch this file, so what is the point of this line ? Maybe have it like this:
```suggestion
export HADOOP_USER_NAME=${HADOOP_USER_NAME:-$(whoami)}
```

Reply:
I think it's a good idea to keep these two files in sync, to keep track of the environment variables we use